source/documentation/upgrade_guide/chap-Updates_between_Minor_Releases.adoc

```diff
@@ -9,6 +9,11 @@ ifdef::context[:parent-context: {context}]
 
 To update from your current version of {vername_rhv} to the latest version of {vername_rhv}, update the {engine-name},  update the hosts, and then change the compatibility version for the cluster, virtual machines, and data center.
 
+[NOTE]
+====
+If upgrading from version 4.4.9 to a later version fails on RHV-H, run the `yum reinstall redhat-virtualization-host-image-update` command to fix the issue.
+====
```

Comment:
please wrap above in rhv-doc. for ovirt it will be:

   If upgrading from version 4.4.9 to a later version fails on oVirt Node, run the `dnf reinstall ovirt-node-ng-image-update` command to fix the issue.